lib/Doctrine/DBAL/Driver/PDOSqlsrv/Statement.php

```diff
@@ -1,37 +0,0 @@
-<?php
-
-declare(strict_types=1);
-
-namespace Doctrine\DBAL\Driver\PDOSqlsrv;
-
-use Doctrine\DBAL\Driver\PDOStatement;
-use Doctrine\DBAL\ParameterType;
-use PDO;
-
-/**
- * PDO SQL Server Statement
- */
-class Statement extends PDOStatement
```

Comment:
Why does this need to be renamed?

Reply:
I just wanted to name it similarly to SQLSrvStatement. It is not subtype of PDOStatement, so I wanted to mark it clearly.